cpp/examples/deck.gl/flight-paths.cc

```diff
@@ -76,11 +76,17 @@ int main(int argc, const char *argv[]) {
 
   auto deckProps = std::make_shared<Deck::Props>();
   deckProps->layers = {createLineLayer(flightDataPath)};
-  deckProps->initialViewState = createInitialViewState();
+  deckProps->initialViewState = createViewState(0.0);
   deckProps->views = {std::make_shared<MapView>()};
   deckProps->width = 640;
   deckProps->height = 480;
 
+  deckProps->onAfterRender = [&deckProps](Deck *deck) {
```

Comment:
This is correct for now. I wonder if this could just be a parameter to `deck.run()` (we'd need to decide if a callback argument was onBeforeRender or onAfterRender)